### PR TITLE
Create Initial ScholarX archive page

### PR DESCRIPTION
**Pull request by dave-v0**
## Purpose

The purpose of this PR is to fix

## Goals

Goal is to build the basic structure of the ScholarX archive page

## Approach

Built the basic structure of the ScholarX archive page

### Screenshots

image

  
### Preview Link

https://pr-1388-sef-site.surge.sh/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: scholarx/archive/archive-v2/styles.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+.filter-container{
+  margin-top:150px;
+  margin-bottom: 200px;
+}
+
+.btn-checkbox {
+  padding-left: 40px;
+  line-height: 1.7;
+}
```
Comment (YoshithaRathnayake) [inline scholarx/archive/archive-v2/styles.css]: Add a new line at the bottom of this file @DilinaBandara

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: success_stories/styles.css [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+.skeleton-storycard-text-details {
+    width: 50px;
+    height: 0.8rem;
+    margin-top: 10px;
+}
+
+.skeleton-storycard-text:last-child {
+    margin-bottom: 0;
+    width: 80%;
+}
```
Comment (YoshithaRathnayake) [inline success_stories/styles.css]: Add a new line at the bottom of this file also @DilinaBandara

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: scholarx/archive/archive-v2/index.html [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+<!--Mustache.js-->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/2.3.0/mustache.min.js"></script>
+<!-- Argon JS -->
+<script src="/assets/js/argon.js?v=1.0.1"></script>
+<!-- Script for static navbar and footer -->
+<script src="/assets/js/navigation.js"></script>
+<!-- External JS file for the JS functions-->
+<script src="functions.js"></script>
+
+
```
Comment (anjula-sack) [inline scholarx/archive/archive-v2/index.html]: Let's remove this extra space

File: scholarx/archive/archive-v2/index.html [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+      </div>
+      <div class="col-6">
+        <div class="input-group">
+            <input type="search" class="form-control rounded search-bar" placeholder="Search" aria-label="Search" aria-describedby="search-addon" />
+            <button type="button" class="btn btn-outline-primary search-btn">search</button>
+          </div>
+      </div>
+    </div>
+</div>
+
```
Comment (anjula-sack) [inline scholarx/archive/archive-v2/index.html]: Here as well

File: success_stories/styles.css [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+
+/* -------------------- Achievements section ---------------------- */
+.story-section ul {
+    font-size: 1rem;
+    font-weight: 300;
+    line-height: 1.8;
+    color: #000000;
+    margin-left: 5px;
+}
+
```
Comment (anjula-sack) [inline success_stories/styles.css]: Here as well

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: success_stories/styles.css (hunk 2 above)
Comment (dave-v0) [reply success_stories/styles.css]: Hi @YoshithaRathnayake  thank you for the comment! As there are a few more changes to be made to the initial structure we are hoping to complete it with @Madhawa97 and then add more changes. Thanks!

File: scholarx/archive/archive-v2/functions.js [hunk 6]
```diff
@@ -0,0 +1,4 @@
+// Write custom js functions for the success stories page here
```
Comment (Madhawa97) [inline scholarx/archive/archive-v2/functions.js]: This comment references the success stories page.

File: success_stories/styles.css (hunk 2 above)
Comment (Madhawa97) [reply success_stories/styles.css]: Hi @DilinaBandara. This CSS file is in the `/success_stories `directory. Is it intentional to add these changes?

File: scholarx/archive/archive-v2/index.html [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+              <li><a class="dropdown-item active" href="#">Action</a></li>
+              <li><a class="dropdown-item" href="#">Another action</a></li>
+              <li><a class="dropdown-item" href="#">Something else here</a></li>
+              <li><hr class="dropdown-divider"></li>
+              <li><a class="dropdown-item" href="#">Separated link</a></li>
+            </ul>
+          </div>
+      </div>
+      <div class="col-6">
+        <div class="input-group">
```
Comment (Madhawa97) [inline scholarx/archive/archive-v2/index.html]:
```suggestion
        <div >
```

Comment (Madhawa97) [reply scholarx/archive/archive-v2/index.html]: The `input-group` class results in the removal of the right border. Therefore, better to remove this and get the `search button` and `input field` into a single row.

**Review by Madhawa97 — CHANGES_REQUESTED** [review]

File: success_stories/styles.css (hunk 2 above)
Comment (anjula-sack) [reply success_stories/styles.css]: we don't need this right? @DilinaBandara @Madhawa97

File: scholarx/archive/archive-v2/styles.css (hunk 1 above)
Comment (anjula-sack) [reply scholarx/archive/archive-v2/styles.css]: Add the end of the line @DilinaBandara

File: scholarx/success-stories/styles.css [hunk 8]
```diff
@@ -205,4 +205,4 @@
     .story-section .highlight {
         font-size: 20px;
     }
-}
+}
```
Comment (anjula-sack) [inline scholarx/success-stories/styles.css]: Add the end of the line

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: success_stories/styles.css (hunk 2 above)
Comment (Madhawa97) [reply success_stories/styles.css]: yes @anjula-sack. If you use any of the styles from `/success_stories/styles.css` you can copy them into the CSS file for this page. @DilinaBandara

Comment (Madhawa97) [reply success_stories/styles.css]: Also, There is another `/success_stories/` directory added outside the `scholarx` directory. @DilinaBandara

Comment (anjula-sack) [reply success_stories/styles.css]: yes, Remove this CSS file

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: scholarx/archive/archive-v2/index.html [hunk 9]
```diff
@@ -0,0 +1,173 @@
...
+            <input class="custom-control-input" id="chkComputerScience" type="checkbox">
+            <label class="custom-control-label" for="chkComputerScience">2020</label>
+        </button>
+        <button class="btn btn-sm custom-checkbox mb-3 btn-checkbox">
+            <input class="custom-control-input" id="chkComputerScience" type="checkbox">
+            <label class="custom-control-label" for="chkComputerScience">2021</label>
+        </button>
+        <button class="btn btn-sm custom-checkbox mb-3 btn-checkbox">
+            <input class="custom-control-input" id="chkComputerScience" type="checkbox">
+            <label class="custom-control-label" for="chkComputerScience">2022</label>
```
Comment (anjula-sack) [inline scholarx/archive/archive-v2/index.html]: All the inputs have the same id

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: scholarx/archive/archive-v2/index.html [hunk 10]
```diff
@@ -0,0 +1,173 @@
...
+          <h6>COHORT</h6>
+          <hr>
+        <div class="mt-2 mb-5">
+          <button class="btn btn-sm custom-checkbox mb-3 btn-checkbox">
+            <input class="custom-control-input" id="chk2019" type="checkbox">
+            <label class="custom-control-label" for="chk2019">2019</label>
+        </button>
+        <button class="btn btn-sm custom-checkbox mb-3 btn-checkbox">
+            <input class="custom-control-input" id="chk2012" type="checkbox">
+            <label class="custom-control-label" for="chk2012">2020</label>
```
Comment (anjula-sack) [inline scholarx/archive/archive-v2/index.html]: chk2020

File: scholarx/archive/archive-v2/index.html [hunk 11]
```diff
@@ -0,0 +1,173 @@
...
+        <div class="dropdown mt-2">
+            <button class="btn btn-secondary dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false">
+              All Universities
+            </button>
+            <ul class="dropdown-menu dropdown-menu-dark">
+              <li><a class="dropdown-item active" href="#">Action</a></li>
+              <li><a class="dropdown-item" href="#">Another action</a></li>
+              <li><a class="dropdown-item" href="#">Something else here</a></li>
+              <li><hr class="dropdown-divider"></li>
+              <li><a class="dropdown-item" href="#">Separated link</a></li>
```
Comment (anjula-sack) [inline scholarx/archive/archive-v2/index.html]: Let's remove this.

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: scholarx/archive/archive-v2/index.html (hunk 10 above)
Comment (dave-v0) [reply scholarx/archive/archive-v2/index.html]: Do I have to remove these two lines?

Comment (anjula-sack) [reply scholarx/archive/archive-v2/index.html]: It says 2012 it should be 2020

File: scholarx/archive/archive-v2/index.html (hunk 9 above)
Comment (Madhawa97) [reply scholarx/archive/archive-v2/index.html]: Reviewing code is not easy. I mean, I didn't think of `id`s. Do you happen to have a checklist or something? @anjula-sack

Comment (anjula-sack) [reply scholarx/archive/archive-v2/index.html]: If this code is ready for your integration. We are good to go. @Madhawa97

**Comment by anjula-sack** [comment]
@DilinaBandara can you take a pull and change the archive-v2 to archive?

**Comment by Madhawa97** [comment]
@anjula-sack I'm ready with the mentees of a mentor algorithm. the data needs some cleaning though.

**Comment by anjula-sack** [comment]
> @anjula-sack I'm ready with the mentees of a mentor algorithm. the data needs some cleaning though.

Great. We can merge this once @DilinaBandara rename the directory. @Madhawa97

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Remove the extra archive directory. It should be scholarx/archive/functions.js

File: scholarx/archive/index.html [hunk 12]
```diff
@@ -0,0 +1,170 @@
...
+                    <div class="pl-4">
+                        <img src="/assets/img/brand/scholarx.png" class="img-fluid align-self-center mb-5 w-50">
+                        <p>ScholarX is an exclusive 6-month program aimed at providing
+                            mentoring support to a selected pool of high Potential undergraduate students
+                            based in Sri Lanka ideally by a Sri Lankan expat currently engaged with one of
+                            the world’s top universities or Fortune 500 companies. It’s our free premium
+                            mentoring platform by Sri Lankans for Sri Lankans working towards creating a culture
+                            of knowledge and expertise sharing without the limitation of geographical borders.</p>
+                    </div>
+
```
Comment (anjula-sack) [inline scholarx/archive/index.html]:
```suggestion             
```

**Review by anjula-sack — APPROVED** [review]
Good job! @DilinaBandara

**Pull request merged**